nautobot/utilities/testing/views.py

```diff
@@ -16,6 +16,7 @@
 from nautobot.extras.models import Tag
 from nautobot.users.models import ObjectPermission
 from nautobot.utilities.permissions import resolve_permission_ct
+from nautobot.utilities.fields import JSONArrayField
```

Comment:
Do we still need `from django.contrib.postgres.fields import ArrayField` on line 6 above, or can/should it be removed?